frontend/public/components/_resource.scss

```diff
@@ -103,6 +103,10 @@
   background-color: $color-ingress-dark;
 }
 
+.co-m-resource-virtualmachineinstance {
```

Comment:
we should put this into kubevirt plugin - maybe `src/style.scss` ?

Reply:
:+1: thanks, make sense to me too !